lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -1095,6 +1084,23 @@ sub _start_synapse
    })
 }
 
+sub generate_listeners
+{
+   my $self = shift;
+
+   return
+      {
+         type         => "http",
+         port         => $self->{ports}{synapse_unsecure},
+         bind_address => "127.0.0.1",
+         x_forwarded  => JSON::true,
+         resources    => [{
+            names => [ "client", "federation", "replication" ]
```

Comment:
This drops "metrics" since it has a separate listener?

Reply:
that, and the fact that we just don't use the metrics in sytest.